python/ray/_private/tracing_utils.py

```diff
@@ -0,0 +1,72 @@
+import inspect
+import os
+from typing import Callable, List
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter, SimpleSpanProcessor
+from ray._private.utils import import_attr
+from ray.serve.schema import TracingConfig
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+
+
+DEFAULT_TRACING_EXPORTER_IMPORT_PATH = (
+    "ray._private.tracing_utils:default_tracing_exporter"
+)
+
+
+def default_tracing_exporter() -> List[SimpleSpanProcessor]:
+    os.makedirs("/tmp/spans", exist_ok=True)
+    return [
+        SimpleSpanProcessor(
+            ConsoleSpanExporter(out=open(f"/tmp/spans/{os.getpid()}.json", "a"))
+        )
+    ]
+
+
+def validate_tracing_exporter(func: Callable) -> None:
+    if inspect.isfunction(func) is False:
+        raise TypeError("Tracing exporter must be called on a function.")
+
+    if not isinstance(func, (Callable, str)):
```

Comment:
this is weird, the type annotation above says just callable

Reply:
Fixed validation